examples/gwlb_with_vmseries/.header.md

```diff
@@ -0,0 +1,62 @@
+# VM-Series Azure Gateway Load Balancer example
+
+The exmaple allows to deploy VM-Series firewalls for inbound and outbound traffic inspection utilizing Azure Gateway Load Balancer in service chain model as described in the following [document](https://docs.paloaltonetworks.com/vm-series/10-2/vm-series-deployment/set-up-the-vm-series-firewall-on-azure/deploy-the-vm-series-firewall-with-the-azure-gwlb).
+
+## Usage
+
+### Deployment Steps
+
+* Checkout the code locally.
+* Copy `example.tfvars` to `terraform.tfvars` and adjust it to your needs.
+* Copy `files/init-cfg.txt.sample` to `files/init-cfg.txt` and fill it in with required bootstrap parameters (see this [documentation](https://docs.paloaltonetworks.com/vm-series/10-2/vm-series-deployment/bootstrap-the-vm-series-firewall/create-the-init-cfgtxt-file/init-cfgtxt-file-components) for details).
+* (optional) Authenticate to AzureRM, switch to the Subscription of your choice if necessary.
+* Initialize the Terraform module:
+
+      terraform init
+
+* (optional) Plan you infrastructure to see what will be actually deployed:
+
+      terraform plan
```

Comment:
this notation is less error prone when doing conversions from MD to MDX/Docusaurus 

````suggestion
  ```bash
  terraform plan
  ```
````

Reply:
Fixed